tuning/trackers/filelogging_tracker.py

```diff
@@ -56,7 +56,9 @@ def _track_loss(self, loss_key, log_name, log_file, logs, state):
             log_obj = {
                 "name": log_name,
                 "data": {
-                    "epoch": round(logs["epoch"], 2),
+                    "epoch": round((logs["epoch"] - 1), 2)
```

Comment:
Also I know the issue was for this to specifically be in the training logs to start at 0 indexing, but the loss that is printed out will still start at 1 correct? Since this change would only affect the logging and not how tranformers is calculating and logging the epochs

Reply:
Yes. I didn't find any argument or function which makes the transformer epoch to start from 0 instead of 1. This change is just a change in log file, but transformers will still start from 1.

Reply:
Also, I do have asked Mynhardt Burger, regarding clarification, if they just expects changes in epoch-index in the logging file of fms-hf-tuning repo or does it expects to change the transformers calculations and working to start from epoch 0 ?
[Redhat issue](https://issues.redhat.com/browse/RHOAIENG-10792)